SampleApp/XAMLator.SampleApp/LabelView.xaml.cs

```diff
@@ -1,10 +1,8 @@
-﻿using System;
-using System.Collections.Generic;
-using Xamarin.Forms;
+﻿using Xamarin.Forms;
 
 namespace XAMLator.SampleApp
 {
-	public partial class LabelView : ContentView
+    public partial class LabelView : ContentView
```

Comment:
It seems the indentation has been changed. I will check into exporting the current setting into an editorconfig file